metals/src/main/scala/scala/meta/internal/metals/ScalafixProvider.scala

```diff
@@ -242,6 +242,8 @@ case class ScalafixProvider(
           |OrganizeImports.removeUnused = true
           |OrganizeImports.targetDialect = Scala3
           |
+          |ExplicitResultTypes.fetchScala3CompilerArtifactsOnVersionMismatch = true
```

Comment:
https://github.com/scalacenter/scalafix/commit/c2afbcc145c292e1599ba8ef337ee0334f58d456

Reply:
Users of ExplicitResultTypes will most likely have a custom cone anyway, but I guess it doesn't hurt to put it there